types/address_test.go

```diff
@@ -133,6 +145,52 @@ func (s *addressTestSuite) TestRandBech32AccAddrConsistency() {
 	s.Require().Equal(types.ErrEmptyHexAddress, err)
 }
 
+func FuzzBech32AccAddrConsistencyYAML(f *testing.F) {
```

Comment:
please move fuzzer tests to to a address_fuzz_test.go file. I always find it confusing to see them with others.

Reply:
Done